HHS/Head-Start-TTADP
# Show tta overview widget to admins only

## Description of change
We got a request to turn off the tta overview widge to analyze the statistics. This PR enables viewing of the TTA overview widget for admins only.


## How to test
- This can be tested on dev 
- Navigate to https://tta-smarthub-dev.app.cloud.gov/activity-reports
- Before disabling admin access, make sure someone with an admin access is available to re-enable it (please slack me a request if needed)
- Go to admin UI
- Uncheck admin access
- Navigate to activity reports
- Refresh the page
- Note that the widget doesn't show

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143


## Checklists

### Every PR


- [n/a] Meets issue criteria
- [n/a] JIRA ticket status updated
- [n/a] Code is meaningfully tested
- [ n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ n/a] API Documentation updated
- [n/a ] Boundary diagram updated
- [n/a ] Logical Data Model updated
- [ n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
